extractor/service.go

```diff
@@ -1,20 +1,250 @@
 package extractor
 
 import (
+	"context"
+	"encoding/json"
+	"errors"
 	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	log "github.com/inconshreveable/log15"
+
+	"github.com/bblfsh/sdk/protocol"
+	"google.golang.org/grpc"
+
+	"gopkg.in/src-d/go-git.v4"                 // git.Open
+	"gopkg.in/src-d/go-git.v4/plumbing"        // Hash, Repository
+	"gopkg.in/src-d/go-git.v4/plumbing/object" // object.File
+	"gopkg.in/src-d/go-git.v4/storage"
```

Comment:
We use just 3 groups of imports: stdlib, internal, external.
Also avoid these kind of comments, they get outdated easily. Anyway, since everything is qualified with package name, it's easy to track what the package is used for.